autoload/lsp.vim

```diff
@@ -697,3 +698,29 @@ endfunction
 function! lsp#complete(...) abort
     return call('lsp#omni#complete', a:000)
 endfunction
+
+let s:event_queue = []
+let s:event_timer = -1
+
+function! lsp#add_event_queue(queue) abort
+    for l:queue in s:event_queue
+        if l:queue[0] == a:queue[0] && l:queue[1] == a:queue[1]
+            return
+        endif
+    endfor
+    call add(s:event_queue, a:queue)
+    call lsp#log('s:send_event_queue() will be triggered')
+    call timer_stop(s:event_timer)
+    let lazy = &updatetime > 1000 ? &updatetime : 1000
+    let s:event_timer = timer_start(lazy, function('s:send_event_queue'))
+endfunction
+
+function! s:send_event_queue(timer) abort
```

Comment:
I remember in some version of neovim `timer` as arg was not present. One way to get around it would be to use `function! s:send_event_queue(...) abort` instead.

Reply:
Ah, sorry. I was confused. Will fix.